components/UI/navbar.tsx

```diff
@@ -44,7 +47,15 @@ const Navbar: FunctionComponent = () => {
     process.env.NEXT_PUBLIC_IS_TESTNET === "true" ? "testnet" : "mainnet";
   const [txLoading, setTxLoading] = useState<number>(0);
   const [showWallet, setShowWallet] = useState<boolean>(false);
-  const { data: profile } = useStarkProfile({ address });
+  const [profile, setProfile] = useState<StarkProfile | undefined>(undefined);
+  const { starknetIdNavigator } = useContext(StarknetIdJsContext);
+
+  // could be replaced by a useProfileData from starknet-react when updated
+  useEffect(() => {
+    if (starknetIdNavigator !== null && address !== undefined) {
+      starknetIdNavigator.getProfileData(address).then(setProfile);
+    }
+  }, [address]);
```

Comment:
Because starknet react is broken with the new contract ? If so then we need to create an issue cause it also means starknet js is broken. 

I forgot about it :/